lib/govuk_app_config/govuk_error.rb

```diff
@@ -41,6 +41,7 @@ def self.configure
     Sentry.init do |sentry_config|
       config = Configuration.new(sentry_config)
       # Avoid "Sending envelope with items ... to Sentry" logspew on stdout.
+      config.logger = Logger.new($stdout)
```

Comment:
I don't think we want to do this as we'll lose things like structured logging or the directing to log files in development or test. I think it's right that this logger is the application one.

I'm just going to have a ponder about your problem though

Reply:
Hm that's a fair point. How sure are you that these logs are honouring those things right now though? 😅 The unwanted logs that I was seeing were unstructured and going to stdout.

Ruby's `Logger` doesn't seem to have a copy constructor or whatever that's called in the Ruby world. What's the usual way to control a gem's log level?

Reply:
Hmm I've had a bit of a look around. Looks like others have found the message annoying in: https://github.com/getsentry/sentry-ruby/issues/1917

I'm wondering what are sessions? and is it even something we want? I guess they're an indication of data being sent to Sentry. I'm not sure why we're sending data when not erroring 🤔 

I feel like a way we could do your change without things behaving too weirdly is if we did something like

```
config.logger = config.logger.dup
config.logger = Sentry::Logger::WARN
```

It'd still be a bit icky as we'd have a rogue extra logger in the app that wouldn't behave right if you change the global logger.

As an aside I think we may have this code in the wrong place, as this is configuration it should be done in https://github.com/alphagov/govuk_app_config/blob/7a4f323d9728cccbdd014e6209a48ab30db09753/lib/govuk_app_config/govuk_error/configuration.rb#L15 with the other configuration for Sentry.

Reply:
Looking at this: https://docs.sentry.io/platforms/ruby/configuration/releases/#sessions I kinda think we can just turn session tracking off. I don't think we even know we have it let alone use it. 

I know it's a bit of a weird way to solve this problem but I think it's a part of a bunch of extra monitoring that Sentry offers which we don't make any use of.

Reply:
Sorry @sengi I totally hadn't noticed your reply in amongst my spiel.

Yeah `.dup` would probably be the thing you want I reference it a bit earlier.

Reply:
Heh, `dup` on an object which [doesn't provide its own implementation of `dup`](https://github.com/ruby/logger/search?q=dup) and manages non-[POD](https://en.wikipedia.org/wiki/Passive_data_structure) resources like files sounds scary to me 😅

If we don't use Sentry's session tracking then I'm happy to go with just turning that off like you suggest 👍

Reply:
No worries, GitHub seems to be a bit erratic at updating sometimes 😅

I've removed the ill-fated attempt at controlling the log level and turned off the auto sessions feature as you suggested (in the right place hopefully this time - thanks for that!)